gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/_shared_macros.j2

```diff
@@ -0,0 +1,69 @@
+{#
```

Comment:
This is a regular file, not a symlink:

```
git ls-files -s  gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/_shared_macros.j2
100644 7094d3039af53ead4b844ba9952d7084f0526bd2 0       gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/_shared_macros.j2
```
See [the meaning of Git modes](https://git-scm.com/book/sv/v2/Git-Internals-Git-Objects#:~:text=Simple%20version%20of%20the%20Git%20data%20model.&text=In%20this%20case%2C%20you're,which%20specifies%20a%20symbolic%20link.). (Credit: [SO answer](https://git-scm.com/book/sv/v2/Git-Internals-Git-Objects#:~:text=Simple%20version%20of%20the%20Git%20data%20model.&text=In%20this%20case%2C%20you're,which%20specifies%20a%20symbolic%20link.))

Reply:
Good catch and thanks for the link! Fixed in https://github.com/googleapis/gapic-generator-python/pull/1999/commits/7957dedeb97c3366f922a6a9805433b130fad47b
```
(py39) partheniou@partheniou-vm-3:~/git/gapic-generator-python$ git ls-files -s  gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/_shared_macros.j2
120000 6884f2def889f3f4f2a5b19b54e76a52c2826174 0       gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/_shared_macros.j2
```

Reply:
I reverted this change in https://github.com/googleapis/gapic-generator-python/pull/1999/commits/542e4e194d44ee8d1d2d3a8b43848b8758ba42d3 and filed https://github.com/googleapis/gapic-generator-python/issues/2028 to investigate if symlinks can be used.

Reply:
Oh, I see. You reverted back to the duplicate file, like I said in that last paragraph. SG. Could you add a note that this is intended to be a symlink to the other file, and for now is a duplicate?